ledger/participant-state/kvutils/src/test/lib/scala/com/daml/ledger/participant/state/kvutils/ParticipantStateIntegrationSpecBase.scala

```diff
@@ -567,28 +570,48 @@ abstract class ParticipantStateIntegrationSpecBase(implementationName: String)(
       val partyNames =
         partyIds
           .map(i => Ref.Party.assertFromString(s"party-%0${partyIdDigits}d".format(i)))
-          .toVector
+          .to[SortedSet]
 
-      val updatesF = ps
+      val expectedOffsets = partyIds.map(i => toOffset(i)).to[SortedSet]
+
+      val updates = mutable.Buffer.empty[(Offset, Update)]
+      val stateUpdatesF = ps
```

Comment:
Minor: if it was `participantState` I wouldn't need to open more context :)

Reply:
I believe I left this exact same comment when this test was created. 😛